cluster.go

```diff
@@ -37,7 +37,7 @@ func NewClusterFromURL(url, databaseID, token string, timeout time.Duration) (*g
 }
 
 func NewCluster(dialer gocql.HostDialer, username, password string) *gocql.ClusterConfig {
-	cluster := gocql.NewCluster("0.0.0.0") // Placeholder, maybe figure how to make this better
+	cluster := gocql.NewCluster("127.0.0.1") // Placeholder, maybe figure how to make this better
```

Comment:
Could you check the dialer type and, if it's a `gocqlastra.dialer`, pull the host from the bundle? Or perhaps this function could require `gocqlastra`-specific dialer explicitly, which exposes the bundle?
```go
// implemented by gocqlastra.dialer
type AstraDialer interface {
  gocql.HostDialer
  getBundle() *astra.Bundle
}

func NewCluster(dialer AstraDailer, username, password string) *gocql.ClusterConfig {
	cluster := gocql.NewCluster(dialer.getBundle().Host) 
	// ...
}
```

Reply:
That's a good idea but it does introduce a potential point of failure because gocql will attempt to resolve the hostname (for no good reason since these addresses will never be used).